book/guide/hardware-requirements.md

```diff
@@ -0,0 +1,107 @@
+# Hardware Requirements
+
+Currently, the Frankendancer validator uses a lot of components from
+Solana. As such, the hardware requirements are similar to that of [existing
+Solana validators](https://docs.solana.com/running-validator/validator-reqs).
+This page will be updated as and when these requirements change.
+
+## Node Specs
+
+### Validator Node
```

Comment:
If you're going to put tables in here, we can format them as markdown. But I think we can just remove all of this and have a few bullets with the minimum requirements, which will just match the Solana Labs ones.